conan/_internal/api/new/autoools_exe.py

```diff
@@ -1,4 +1,4 @@
-from conan.api.helpers.new.autotools_lib import configure_ac, makefile_am
+from conan._internal.api.new.autotools_lib import configure_ac, makefile_am
```

Comment:
I am not loving that this is highlighted by linters and IDE as a warning of importing something internal.
Is there any workaround for this warning? Maybe relative imports? but those are also not recommended?

Reply:
I can't find anything but double underscore `__internal` that causes no issues with the linters. I can't find any documentation about that double underscore in module names so apparently, it is meaningless.